### DISCUSSION TITLE
Allow multiple pages when creating recipes from images

### DISCUSSION THREAD

**Discussion by counterbeing** [mealie-recipes/mealie#4067] [Feature Request]
### First Check

- [X] I used the GitHub search to find a similar requests and didn't find it.
- [X] Checked the [tasks tagged](https://github.com/mealie-recipes/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

### Please provide a concise description of the problem that would be addressed by this feature.

When taking pictures of recipes to be processed by OpenAI, only one image is accepted, but frequently recipes are more than one page.

### Please provide a concise description of the feature that would resolve your issue.

Allowing additional images to be added.

### Please consider and list out some caveats or tradeoffs made in your design decision

The only tradeoff might be complexity of UX, but I can imagine this could be implemented by adding a button to "add another image"

### Additional Information

- [ ] If this is accepted I'm willing to submit a PR to provide this feature
- [ ] If this is accepted I'm willing to help maintain this feature
- [ ] I'm willing to sponsor/pay a developer to do this work

**Comment by michael-genson** [comment]
Yup, makes sense to me. I built the API to accept multiple images, so if anyone wants to take a crack at this, it _should_ be a purely frontend change

**Reply by counterbeing** [reply]
Awesome! I just tried the single page feature as is, and it's amazing! I'm not likely to pick this up myself any time soon, though am open to it at some point. Free time is tough to come by with young children.

**Comment by jdaviescoates** [comment]
I came here to add exactly this. I just started using Mealie and it's great. But I quite quickly hit this issue with one of my favourite recipes. In the meantime I'll probably edit multiple images together but that's a bit long winded!

(As an aside, I also had to just edit an image because it had multiple short recipes on it and I only wanted one of them - at least I assumed I needed to edit it for it to work - I didn't actually try to see what would happen first!)

**Reply by nununo** [reply]
Regarding your second problem (having multiple short recipes on the same image), Mealie gives you a tool to crop the image before processing it, so no need to crop it beforehand.

**Reply by jdaviescoates** [reply]
Thanks, I'm aware of the crop tool but that doesn't help with images like this where it's just one corner that needs editing out
![IMG_20250422_213840](https://github.com/user-attachments/assets/29f6761a-e7d7-47ab-835a-d9552287a835)

Maybe there was an easier way but I used Canva to block out the unwanted recipe like this 
![20250422_214730_0000](https://github.com/user-attachments/assets/5b9b594f-3a19-4136-adbc-c29abaa2100e)

**Comment by freemaintech** [comment]
Really loving Mealie so far but struggled with importing via image since a lot of recipes are front and back of an index card or multiple pages. +1 for desire to have this feature, hopefully we can see it implemented sooner than later!
